### PR TITLE
create an about page for the documentation

### LINKED ISSUE #187: Add an About page to the documentation
Add and fill an About page to the documentation to provide a high-level description of what simona was designed for.

### PR DESCRIPTION
**Pull request by lararou**
resolve #187

### CODE REVIEW

File: docs/readthedocs/about.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
 (about)=
-# About SIMONA
+# About Simona
 
-## Overview
+Simona is a simulation-tool for generating time-series in the context of distribution grids to be used in grid planning, operation and analysis. The simulation is executed using an agent-based, discrete-event model. Simona is being developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.
```
Comment (danielfeismann) [inline docs/readthedocs/about.md:4]: I would go for SIMONA in capital letters and I think it is simulation tool as well as time-series as separate words.

File: docs/readthedocs/about.md [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 
-SIMONA is an agent-based discrete-event power system simulation model developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.
+## Power Grid Model
+With Simona, you can model your power grid using a broad variety of different available asset types. The input data for the grid must be provided as PSDM-data. (For more information on this data type, go to the [PSDM](https://powersystemdatamodel.readthedocs.io/en/latest/index.html) documentation.) The combination of implemented physical participant models and your provided input data forms a power grid that can be further investigated and analysed using Simona services.
```
Comment (danielfeismann) [inline docs/readthedocs/about.md:7]: I would suggest to have the Hyperlinks also with a floating window (check the link to PSDM at Line Model for example https://simona--199.org.readthedocs.build/en/199/models/line_model.html)

File: docs/readthedocs/about.md [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
-## Further information and Contact Details
+![Basic Simona](images/usersguide/basic_simona_environment.png "Simona System Overview")
 
-Please visit SIMONA Website at [https://simona.ie3.e-technik.tu-dortmund.de](https://simona.ie3.e-technik.tu-dortmund.de) for further Information. There you can find contact details for current developers.
+## Simulation
+Simona is designed to simulate the assets' behaviour with each other and with the grid itself. In this process, Simona uses an individuality approach, meaning that the simulation is focussed on the individual behaviour of the assets rather than the entire grid. 
+
+Since Simona uses an agent-based approach, the assets are modelled as agents. There are system participant agents (SPA), which represent generators or loads commonly used in distribution grids (PV, Wind Turbine, EV, Storages, ...). In addition to that, a grid agent (GA) manages the interaction with other grid agents and with the subordinate SPAs.
+
+![Simona_messages](images/usersguide/agent_messages.png "Simona Agent Messages")
```
Comment (danielfeismann) [inline docs/readthedocs/about.md:16]: I would suggest to have this picture with maybe 60% width to have it smaller...

**Review by danielfeismann — CHANGES_REQUESTED** [review]
Thanks for your work, this looks great. I just have minor comments which could be changed in my point of view.

**Review by staudtMarius — APPROVED** [review]

File: docs/readthedocs/about.md [hunk 4]
```diff
@@ -1,10 +1,33 @@
...
 
-Please visit SIMONA Website at [https://simona.ie3.e-technik.tu-dortmund.de](https://simona.ie3.e-technik.tu-dortmund.de) for further Information. There you can find contact details for current developers.
+## Simulation
+Simona is designed to simulate the assets' behaviour with each other and with the grid itself. In this process, Simona
+uses an individuality approach, meaning that the simulation is focussed on the individual behaviour of the assets rather
+than the entire grid. 
+
+Since Simona uses an agent-based approach, the assets are modelled as agents. There are system participant agents (SPA),
+which represent generators or loads commonly used in distribution grids (PV, Wind Turbine, EV, Storages, ...). In 
+addition to that, a grid agent (GA) manages the interaction with other grid agents and with the subordinate SPAs.
```
Comment (sebastian-peter) [inline docs/readthedocs/about.md:25]: "manages _interactions between subgrids_ and with"

File: docs/readthedocs/about.md [hunk 5]
```diff
@@ -1,10 +1,33 @@
...
+
+<img src="images/usersguide/agent_messages.png" alt="Simona Agent Message" width="60%" height="60%">
+
+
+## Further Information and Contact Details
+
+Please, visit the SIMONA [website](https://simona.ie3.e-technik.tu-dortmund.de) for further Information. You will find
```
Comment (sebastian-peter) [inline docs/readthedocs/about.md:32]: "_There you will also find_"

File: docs/readthedocs/about.md [hunk 6]
```diff
@@ -1,10 +1,33 @@
...
+## Simulation
+Simona is designed to simulate the assets' behaviour with each other and with the grid itself. In this process, Simona
+uses an individuality approach, meaning that the simulation is focussed on the individual behaviour of the assets rather
+than the entire grid. 
+
+Since Simona uses an agent-based approach, the assets are modelled as agents. There are system participant agents (SPA),
+which represent generators or loads commonly used in distribution grids (PV, Wind Turbine, EV, Storages, ...). In 
+addition to that, a grid agent (GA) manages the interaction with other grid agents and with the subordinate SPAs.
+
+<img src="images/usersguide/agent_messages.png" alt="Simona Agent Message" width="60%" height="60%">
```
Comment (sebastian-peter) [inline docs/readthedocs/about.md:27]: Image is broken on the [final page](https://simona--199.org.readthedocs.build/en/199/about.html)

**Review by sebastian-peter — CHANGES_REQUESTED** [review]
Thanks for taking over this PR! I found a bunch of things that need more fixing.

File: docs/readthedocs/about.md [hunk 7]
```diff
@@ -1,10 +1,32 @@
...
-SIMONA is an agent-based discrete-event power system simulation model developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.
+## Power Grid Model
+With SIMONA, you can model your power grid using a broad variety of different available asset types. The input data for
+the grid must be provided in the PowerSystemDataModel format. (For more information on this data model, please visit the
+{doc}`PowerSystemDataModel <psdm:index>` documentation.) The combination of implemented
+physical participant models and your provided input data forms a power grid that can be further investigated and 
+analysed using ancillary tools.
 
-## Further information and Contact Details
+![Basic Simona](images/usersguide/basic_simona_environment.png "SIMONA System Overview")
```
Comment (sebastian-peter) [inline docs/readthedocs/about.md:16]: I think the alt text is the one in the beginning. So I would place the `SIMONA System Overview` text where currently `Basic Simona` is placed.

File: docs/readthedocs/about.md [hunk 8]
```diff
@@ -1,10 +1,32 @@
...
+## Simulation
+SIMONA is designed to simulate the assets' behaviour with each other and with the grid itself. In this process, SIMONA
+uses an individuality approach, meaning that the simulation is focussed on the individual behaviour of the assets rather
+than the entire grid. 
+
+Since SIMONA uses an agent-based approach, the assets are modelled as agents. There are system participant agents (SPA),
+which represent generators or loads commonly used in distribution grids (PV, Wind Turbine, EV, Storages, ...). In 
+addition to that, a grid agent (GA) manages interactions between subgrids and with the subordinate SPAs.
+
+![SIMONA Agent Message](images/usersguide/agent_messages.png)
```
Comment (sebastian-peter) [inline docs/readthedocs/about.md:27]: Alt text could rather be: "SIMONA message protocol overview"

**Review by sebastian-peter — CHANGES_REQUESTED** [review]
Sorry, the alt texts could also be better. That's all though, I think

**Review by sebastian-peter — APPROVED** [review]
Great, thanks!

@danielfeismann You might also have a final look.

**Review by danielfeismann — APPROVED** [review]
Looks good to me, thanks for the improvement of our documentation :)

**Comment by danielfeismann** [comment]
!test

**Pull request merged**